### PR TITLE
✏️ Run cspell to correct spelling errors

### PR DESCRIPTION
**Pull request by JMicheli**
This pull request is pretty straightforward, I used the tool cspell to go through and find spelling errors, which I fixed. It's pretty hard to catch spelling stuff without a spell-checker, so there were a decent few.

All tests still pass and it runs just fine, but probably a good idea to test it a bit to make sure none of the non-comment changes caused an import to fail somewhere.

### CODE REVIEW

File: docs/pages/guides/basics/books.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The following table outlines the supported formats for books in Stump:
...
 | RAR Archive | `.cbr` `.rar` | ✅    | ✅        | ✅   |                                                                                                                                                                                                                                                                                                            |
-| EPUB        | `.epub`       | ✅    | ❌        | ✅   | Epub files aren't generally supported by ODPS-PSE (streaming) unless they are image-only. They are otherwise OPDS-compatible                                                                                                                                                                               |
+| EPUB        | `.epub`       | ✅    | ❌        | ✅   | Epub files aren't generally supported by OPDS-PSE (streaming) unless they are image-only. They are otherwise OPDS-compatible                                                                                                                                                                               |
```
Comment (aaronleopold) [inline docs/pages/guides/basics/books.md:17]: Surprised it got this one

File: packages/browser/src/components/readers/imageBased/paged/AnimatedPagedReader.tsx [hunk 2]
```diff
@@ -38,7 +38,7 @@ const BACKWARD_START_ANIMATION = {
 // funky on mobile still but not nearly as bad.
 // FIXME: animation on mobile without drag looks bad
 // TODO: slow down the animations to test better
-// TODO: kill me, and then make the animations togglable
+// TODO: kill me, and then make the animations toggleable
```
Comment (aaronleopold) [inline packages/browser/src/components/readers/imageBased/paged/AnimatedPagedReader.tsx:41]: lol

File: packages/browser/src/scenes/library/tabs/settings/options/ScannerFeaturesPatchForm.tsx [hunk 3]
```diff
@@ -45,7 +45,7 @@ export default function ScannerFeaturesPatchForm() {
 	return (
 		<Form form={form} onSubmit={handleSubmit} fieldsetClassName="space-y-12">
 			{/* Note: This component doesn't really belong here, but I didn't want to wrap it in its own form when it is just for display */}
-			{/* Should probably create a separate, non-formy variant */}
+			{/* Should probably create a separate, non-form variant */}
```
Comment (aaronleopold) [inline packages/browser/src/scenes/library/tabs/settings/options/ScannerFeaturesPatchForm.tsx:48]: Another funny one

**Review by aaronleopold — APPROVED** [review]
Thanks for doing this! I don't have time to test it right now, but the changes look good to me

File: docs/pages/guides/basics/books.md (hunk 1 above)
Comment (JMicheli) [reply docs/pages/guides/basics/books.md:17]: I added ODPS to the dictionary of accepted words so this one stood out.

**Pull request merged**